aips/aip-x.md

```diff
@@ -0,0 +1,707 @@
+| address        | string          |           | Blockchain address performing the signing.                                                                                            |
+| uri            | string          |           | URI referring to the resource that is the subject of the signing i.e. the subject of the claim.                                       |
+| version        | string          |           | Current version of the message.                                                                                                       |
+| statement      | string          |           | Human-readable ASCII assertion that the user will sign. It MUST NOT contain `\n`.                                                     |
+| nonce          | string          |           | Randomized token to prevent signature replay attacks.                                                                                 |
+| chainId        | string          |           | Identifier for the network locating the address listed separately above.                                                              |
+| issuedAt       | ISO 8601 string |           | `date-time` that indicates the issuance time.                                                                                         |
+| expirationTime | ISO 8601 string |           | `date-time` that indicates when the signed authentication message is no longer valid.                                                 |
+| notBefore      | ISO 8601 string |           | `date-time` that indicates when the signed authentication message starts being valid.                                                 |
+| requestId      | string          |           | System-specific identifier used to uniquely refer to the authentication request.                                                      |
+| resources      | string[]        |           | List of information or references to information the user wishes to have resolved as part of the authentication by the relying party. |
+
+### AptosSignInMessage
+
+Once the fields are provided from the application, the wallet will be responsible for constructing the SIWA message. If the mandatory fields are not provided, they must be populated by the wallet.
+
+| Name                  | Type            | Mandatory | Description                                                                                                                           |
+| --------------------- | --------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| fields.domain         | string          | ✓         | `dnsauthority` that is requesting the signing                                                                                         |
```

Comment:
From the text, it is not clear: is `AptosSignInMessage::fields.domain` set to `AptosSignInInput::domain`?

I am a bit confused as to why you are you re-stating the same fields here.

Is there a less redundant way to describe this? May need to meet to clarify.